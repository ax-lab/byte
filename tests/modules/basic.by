print 'basic module'

let simple = load mod_simple

print 'the simple answer is', simple.ANSWER
