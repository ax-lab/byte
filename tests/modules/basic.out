basic module
simple module
the simple answer is 42
